src/en/caas.md

```diff
@@ -0,0 +1,145 @@
+Title: CAAS and Juju
+TODO:  Once 2.4 is officially released remove support Note and update status output
+
+# CAAS and Juju
+
+CAAS is *Containers as a Service*, a cloud service that allows users to upload,
+organize, run, scale, manage, and stop containers. Combining this with Juju
+opens up a new vista of possibilities for Juju users. Currently, Juju supports
```

Comment:
I'm not sure about a 'vista of possibilities', let's just stick with practical benefits